twindle-cli/index.js

```diff
@@ -9,19 +9,28 @@ const { sendToKindle } = require("./utils/send-to-kindle");
 async function main() {
   prepareCli();
 
-  const { format, outputFilename, tweetId, kindleEmail } = getCommandlineArgs(process.argv);
+  const {
+    format,
+    outputFilename,
+    tweetId,
+    kindleEmail,
+    mock,
+  } = getCommandlineArgs(process.argv);
 
   try {
-    const tweets = await getTweetsFromTweetId(tweetId);
+    // this next line is wrong
+    let tweets = require("./twitter/twitter_responses/response-version2-tweetthread.json");
```

Comment:
@Mira-Alf please advise on the correctness of this code, which mock json to use.